deploy/olm-catalog/ibm-block-csi-operator-community/1.10.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml

```
@@ -229,6 +229,44 @@ spec:
           - description: The current version of the driver.
             displayName: Version
             path: version
+      - name: hostdefiners.csi.ibm.com
```

```suggestion
      - name: hostdefiners.block.csi.ibm.com
```
hope it's not too late

---

but our group is `csi.ibm.com` not `block.csi.ibm.com`

---

can't the host definer group be different than the driver/operator group?

---

why? we can try but it requires new files to set up, I feel that all controllers should be under the same group because the group doesn't mean much, and I don't see a big difference between `block.csi.ibm.com` and `csi.ibm.com`

---

I think there is a difference, because ibm has other csi drivers, e.g. [scale](https://github.com/IBM/ibm-spectrum-scale-csi/blob/v2.6.0/operator/controllers/config/constants.go#L58)
(btw, it seems [vpc use `.io`](https://github.com/IBM/ibm-vpc-block-csi-driver-operator/blob/d07f5fe0a21b4d6611a1b6090d58e4001fc34440/pkg/util/const.go#L11), hope it's not because of us)

---

I am not sure we have time to change it

---

:(